guide/interactions/replying-to-slash-commands.md

```diff
@@ -0,0 +1,187 @@
+# Replying to slash commands
+
+Discord provides users the option to create client-integrated slash commands. In this section you will be learning how to respond to these commands using discord.js!
+
+::: tip
+You need to have at least one slash command set-up for your application to follow the instructions on this page. If you haven't done that yet, refer to [the previous page](/interactions/registering-slash-commands/).
+:::
+
+## Receiving interactions
+
+Every slash command is an `interaction`, so to respond to a command you need to setup an event listener that will execute code when your application receives an interaction:
+
+```js
+client.on('interaction', interaction => {
+	console.log(interaction);
+});
+```
+        
+However, not every interaction is a slash command. Let's make sure to only receive slash commands by making use of the `interaction#isCommand()` method:
```

Comment:
A word on what else could be an interaction maybe?

Reply:
currently there are no other public interaction types available, will revisit once that changes

Reply:
This has been adressed